apps/nowcasting-app/components/helpers/utils.ts

```diff
@@ -573,3 +574,26 @@ export const createBucketObject: (
     increment
   };
 };
+export function calculateChartYMax(
+  chartData: ChartData[],
```

Comment:
worth a function description in here?, to make it clear what maxY does. 

In some ways this in maxY is avtually mininum, I think its the minimum the value that is returned